README.md

```diff
@@ -1,2 +1,73 @@
-# template-pkg
-A template repository for creating a Toit package
+# Package template
+
+A template repository for creating a Toit package.
+
+## Toit package
+Use `toit.pkg describe` or `toit pkg describe` (depending on which Toit
+variant you use) to see how https://pkg.toit.io extracts package
+information.
+
+Either add a `name: ...` entry to the package.yaml or change the title
+of this README to the package name.
+
+Either add a `description: ...` entry to the package.yaml or ensure
+that the first paragraph of this README can be used as a description.
+
+## Copyright
+Don't forget to update the copyright holder in the license files.
+There are (up to) three license files:
+- `LICENSE`: usually MIT
+- `examples/EXAMPLES_LICENSE`: usually BSD0
+- `tests/TESTS_LICENSE`: usually BSD0
+
+We recommend to use the following Copyright header in `src` files (with your
```

Comment:
Maybe mention that the package code goes in the `src` folder and that that folder needs to be added. As I mentioned in a comment, I would prefer to have the `src` folder around as part of the template.

Reply:
done.